webhooks/mutatepod_test.go

```diff
@@ -14,6 +14,7 @@ import (
 
 var _ = Describe("GCPWorkloadIdentityMutator.mutatePod", func() {
 	var m *GCPWorkloadIdentityMutator
+	var defaultMode int32 = 777
```

Comment:
nits: `777` is too wide even in the test (not realistic). Could you use `0444` here?

```suggestion
	var defaultMode int32 = 0444
```

Reply:
I know it isn't realistic, but I wanted to have a different value from the default. I'll set 400 instead of 777 (something different from default)